### PR TITLE
Better OLM package generation

### PR DESCRIPTION
**Pull request by razvan**
This is part of the Webhook spike on OpenShift. TODO: add issue here.

### CODE REVIEW

File: olm/build-bundles.sh [hunk 1]
```diff
@@ -1,40 +1,77 @@
...
-      esac
-      shift
-  done
-
-  # e.g. "airflow" , "spark-k8s"
-  OPERATOR=$(basename "${OPERATOR_NAME}" | rev | cut -d- -f2- | rev)
+	INITIAL_DIR="$PWD"
+
+	VERSION=""
+	BRANCH=""
```
Comment (adwk67) [inline olm/build-bundles.sh:49]: is this ignored completely? If so we can remove it.

**Review by adwk67 — CHANGES_REQUESTED** [review]
Just some minor things.

File: olm/build-bundles.sh (hunk 1 above)
Comment (razvan) [reply olm/build-bundles.sh]: Done: https://github.com/stackabletech/stackable-utils/pull/61/commits/2328728e8fe57cad2da156dd1a6429414f1546bc

**Review by adwk67 — APPROVED** [review]
lgtm.

**Pull request merged**